index.html

```diff
@@ -0,0 +1,339 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <meta http-equiv="X-UA-Compatible" content="ie=edge" />
+    <title>Device</title>
+    <link rel="stylesheet" href="css/style.css" />
+  </head>
+  <body>
+    <header class="header">
+      <a href="/index.html"><h1 class="logo">DEVICE</h1></a>
+      <div class="header__user">
+        <ul class="user__list">
+          <li>
+            <img src="img/svg/search.svg" alt="search" />
+            <p>Поиск</p>
+          </li>
+          <li>
+            <img src="img/svg/user.svg" alt="user" />
+            <p>Войти</p>
+          </li>
+          <li>
+            <img src="img/svg/compare.svg" alt="compare" />
+            <p>Сравнить</p>
+          </li>
+          <li>
+            <img src="img/svg/cart.svg" alt="car" />
+            <p>Корзина</p>
+          </li>
+        </ul>
+      </div>
+      <nav class="header__nav">
+        <ul class="nav__list">
+          <li>
+            Каталог товаров
+            <img src="img/png/plus.png" alt="plus" class="plus" />
+          </li>
+          <li>Доставка</li>
+          <li>Гарантия</li>
+          <li>Контакты</li>
+        </ul>
+      </nav>
+      <div class="header__01">
+        <p class="numbers">01</p>
+      </div>
+      <section class="footer__catalog">
+        <h2 class="hidden">Каталог</h2>
+        <ul class="catalog__list">
+          <li>Виртуальная реальность</li>
+          <li>Моноподы</li>
+          <li>Экшн камеры</li>
+          <li>Фитнес браслеты</li>
+          <li>Умные часы</li>
+          <li>Квадрокоптеры</li>
+        </ul>
+      </section>
+    </header>
+    <main class="main">
+      <section class="main__content">
+        <div class="main__content__image">
+          <h2 class="hiden">Самая длинная палка для селфи</h2>
+          <a href=""><img src="img/png/slider-1.png" alt="selfy stick"/></a>
+        </div>
+        <div class="main__content__description">
+          <p>Делай селфи, как Бен Стиллер!</p>
+          <p>
+            Самая длинная палка для селфи доступна в нашем магазине.
+            Восемь(Восемь, Карл!) метров длиной и весом всего 5 килограмм.
+          </p>
+          <p>подробнее</p>
+          <button
+            class="Button__Next__1"
+            aria-label="Следующий слайд"></button>
+          <button
+            class="Button__Next__2"
+            aria-label="Следующий слайд"></button>
+          <button
+            class="Button__Next__3"
```

Comment:
what's is the name of classes? "promo-slider__advantages-col--third" is better
don't name classes like that.
button-next-fist eventually